runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingSpotOverlay.java

```diff
@@ -73,6 +78,17 @@ public Dimension render(Graphics2D graphics)
 			}
 
 			Color color = npc.getGraphic() == GraphicID.FLYING_FISH ? Color.RED : Color.CYAN;
+
+			if (config.showMinnowOverlay())
```

Comment:
What happens if you also have showIcons enabled

Reply:
right now if showIcons is enabled, the seconds text will appear above the icon, however if disabled the seconds and 'Minnows' text overlap so I will try figure something out